homeassistant/components/alexa/entities.py

```diff
@@ -645,6 +649,24 @@ def interfaces(self) -> Generator[AlexaCapability]:
         yield Alexa(self.entity)
 
 
+@ENTITY_ADAPTERS.register(remote.DOMAIN)
+class RemoteCapabilities(AlexaEntity):
+    """Class to represent Remote capabilities."""
+
+    def default_display_categories(self) -> list[str]:
+        """Return the display categories for this entity."""
+        return [DisplayCategory.REMOTE]
+
+    def interfaces(self) -> Generator[AlexaCapability]:
+        """Yield the supported interfaces."""
+        yield AlexaPowerController(self.entity)
+        yield AlexaModeController(
+            self.entity, instance=f"{remote.DOMAIN}.{remote.ATTR_ACTIVITY}"
+        )
```

Comment:
I am not sure about the mode controller. It does not seem to be picked up. (But I may be wrong),

Reply:
I am not sure if i understand correctly what you mean. 
It is definitely being used for the remote domain. I tested it, both with postman and Alexa itself and it is working just fine.